data/projects.yml

```
@@ -0,0 +1,12 @@
+title: Projects
+introduction: |
+  These are the open source projects which are officially hosted by the
```

```suggestion
  These are the open-source projects which are officially hosted by the
```
see open-source as an adjective according to merriam-webster:
https://www.merriam-webster.com/dictionary/open-source

---

Our current practice is to write it without the hyphen, especially when it's about terms which acts more as names such as "open source project".